E3Next/Processors/Zoning.cs

```diff
@@ -31,6 +31,8 @@ public static void Zoned(Int32 zoneId)
             {
                 CurrentZone = new Zone(zoneId);
                 ZoneLookup.Add(zoneId, new Zone(zoneId));
+            } else {
```

Comment:
the out param of trygetvalue already does this. 
if it doesn't find it it adds and then sets current zone.

Reply:
I see, sorry for the noise. 

![zoning](https://user-images.githubusercontent.com/1665885/209863548-1a27c655-2579-4d0f-84ea-3138364a9288.PNG)

I was running into an issue where Zoning.CurrentZone.ShortName wasn't returning the correct value. Looks like upon further inspection it was because there were still instructions it was processing after the zoneIn, so when I was trying to read from Zoning.CurrentZone.ShortName, the update hadn't happened yet (in the screenshot, it takes 9 seconds after for Zoning.Zoned to be called and reset the value)